quick-install/nginx-deployment.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
```

Comment:
Can we delete the comment? I think it will cause confusion as 1.9 is so old it shouldn't be being used any more.

Reply:
Also, I wonder if we're better off using a one-line kubectl command? This basically just does:
```
kubectl create deploy trow-test --image=trow.default/test/nginx:apline
```

Reply:
I will delete this file, committed by accident